third-party/rsutils/include/rsutils/string/ip-address.h

```diff
@@ -19,33 +20,50 @@ namespace string {
 //
 class ip_address
```

Comment:
Is string the right place for this class? Is this only a string manipulation helper?

Reply:
Good question, I was wondering myself.
It was put there because it was originally a string representation if an IP address. In a way, it still is.
I didn't want to make extensive changes everywhere so I left as-is. Refactoring this is possible, but then I think it belongs in another commit, and we have to choose: where would you put it?